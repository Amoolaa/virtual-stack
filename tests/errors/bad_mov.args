"tests/errors/bad_mov.x2017"
