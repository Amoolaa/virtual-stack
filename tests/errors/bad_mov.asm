FUNC LABEL 0
    MOV STK A VAL 9
    MOV VAL 6 STK A
    PRINT STK A
    RET
