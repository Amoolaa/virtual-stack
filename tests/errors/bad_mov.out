MOV: destination cannot be value typed.
